readme/installation.md

```diff
@@ -0,0 +1,105 @@
+# Installation
+
+Below are installation instructions for certain distributions.  If your
+distribution is not included, use the <a
+href="#manual-installation">Manual installation</a> instructions.
+
+We welcome contributors to maintain missing distributions.
+
+Please note that the distribution package may lag with the latest bug
+fixes.  Use the <a href="#manual-installation">Manual installation</a>
+method to get the latest bug fixes.
+
+## Manjaro/Arch
+
+```bash
+yay -S input-remapper-git
+sudo systemctl restart input-remapper
+sudo systemctl enable input-remapper
+```
+
+## Ubuntu/Debian
+Get the latest version:
+
+```bash
+sudo apt install git python3-setuptools gettext
+git clone https://github.com/sezanzeb/input-remapper.git
+cd input-remapper && ./scripts/build.sh
+sudo apt install -f ./dist/input-remapper-2.0.0.deb
+```
+
+Or alternatively, use one of the following methods.
+
+### Install using a .deb file
+
+1. Download the `.deb` file from the
+[release page](https://github.com/sezanzeb/input-remapper/releases)
+2. and install it manually replacing `XXX` with the current version:
+```bash
+sudo apt install -f /path/to/input-remapper-XXX.deb
```

Comment:
Whenever I make a new release, I do a global search for the old version and replace the version-string everywhere, so there is no need for the XXX placeholder.